lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -1,31 +1,43 @@
 # Welcome to Serverless!
 #
+# This file is the main config file for your service.
+#
 # For full config options, check the docs:
 #    docs.serverless.com
 #
 # Happy Coding!
+service: execute-on
 
-service: execute-ingest  # NOTE: update this with your service name
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+# frameworkVersion: "=X.X.X"
 
+# plugins:
 plugins:
+  # install serverless pseudo parameters using the following:
+  #   1) $ npm install serverless-pseudo-parameters --save-dev
   - serverless-pseudo-parameters
+  # Add simple-ssh as part of plugins so that serverless function shall package this module and deploy to aws
+  - simple-ssh
+  - lodash
```

Comment:
`lodash` and `simple-ssh` aren't serverless plugins, but they are dependencies of the lambda function. They should be listed in `package.json` and installed with `npm install`.